doc/dev/release.md

```diff
@@ -216,6 +234,11 @@ Call the script with the only argument being the new SDK version:
 $ ./release.sh v1.3.0
 ```
 
+> **NOTE** To run the above script from an OSX env you will need to: 
+> - Use gnu-sed instead of the default sed for the script work. Install it with `brew uninstall gnu-sed` 
+> - Export the gnu-sed to use it as "sed" in your bash profile. (E.g `export PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$PATH"`)
+> - Ensure that you will be using the GNU make instead of the default provide one by installing it with `brew install make`. Note that you need to add the new path in the bash profile as well. (E.g `export PATH="/usr/local/opt/make/libexec/gnubin:$PATH"`)
```

Comment:
There should be a better way than modifying `PATH` for these two commands. Is `gnu-sed` not added to `PATH` after installation, ex. in `/usr/local/bin`?

Reply:
No, it is not added by default. The brew will add the output the msg but to `export PATH="/usr/local/opt/make/libexec/gnubin:$PATH"` but IMO it is important to highlight here since it is someone just miss it.

Reply:
I'd rather not see "here's how to modify your shell environment" in a release doc, as different shells are used across the dev community. I'd rather see something like:
> Make sure you install GNU `sed` and `make` as you cannot use OSX defaults, then ensure they are present in your $PATH as `sed` and `make`, respectively.

Reply:
Since it is specific for OSX I' like to describe the steps to the install, however, I improve the text with your suggestion a well.